src/symbols/Node/Node.tsx

```diff
@@ -241,6 +240,14 @@ export const Node: FC<Partial<NodeProps>> = ({
     }
   });
 
+  useEffect(() => {
```

Comment:
Can you provide some context on this change?

Reply:
When dragging, checkNodeLinkable is working for each node on the tree, with the condition "enteredNode?.id === properties.id" - only for the node, that is the drop target